### DISCUSSION TITLE
Is there any API.

### DISCUSSION THREAD

**Discussion by SohamTilekar** [sharkdp/numbat#540] [Q&A]
I am Working on a Project, In Which I can give the Expression which is Shown in the CLI Image in the ReadMe.md & I want the Answer.

**Comment by sharkdp** [comment]
You can use the `numbat` crate as a library: https://docs.rs/numbat/. To be honest, we haven't spent much time making this a great experience. But it's definitely possible and there are a few examples that use Numbat like this.

**Reply by SohamTilekar** [reply]
> You can use the `numbat` crate as a library: https://docs.rs/numbat/. To be honest, we haven't spent much time making this a great experience. But it's definitely possible and there are a few examples that use Numbat like this.

Actually I want to call this from the Python Therefore I need something like API.

**Reply by sharkdp** [reply]
There is currently no Python API for Numbat.
